source/default/_patterns/00-protons/demo/spacing.twig

```diff
@@ -3,12 +3,15 @@
     <div class="col mt-sm-5">
 
       <h1>Spacing</h1>
-      <p class="lead">To add to these items, adjust <code>$spacing()</code> in <code>_variables.scss</code></p>
-      {% for spacing, value in scssSpacing %}
-        <div style="margin: 10px;">
-          <div style="height: {{ value }}; width: {{ value }}; outline: solid 1px black; display: inline-block; margin-right: 5px;"></div>
-          <span>{{ value }} : <code>$spacing({{ spacing }})</code></span>
-        </div>
+      <p class="lead">To add to these items, adjust <code>$spacers()</code> in Bootstrap's <code>_variables.scss</code>.</p>
```

Comment:
If we're using the Bootstrap vars as default, I think we should update from `<code>_variables.scss</code>.</p>` to `<code>_bootstrap_variables.scss</code>.</p>`? CC: @illepic

Reply:
Yes! If I'm following how this is changed, I think that's where spacing overrides would now live by using Bootstrap's sizing vars.

Reply:
Yep! I think this is good to merge once that's updated to `non-printing/_bootstrap-overrides.scss`

Reply:
Awesome. Thanks for clarifying!